modules/cnv-about-dedicated-resources.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-about-dedicated-resources_{context}"]
+
+= About dedicated resources
+
+When you enable dedicated resources for your virtual machine, your virtual
+machine's workload is scheduled on CPUs that will not be used by other
+processes. By using dedicated resources, you improve the performance of the
+virtual machine and the accuracy of latency predictions.
+
+To prepare to enable dedicated resources, the `cpumanager` policy on the node
+must be `static`. When the policy is `static`, the `cpumanager` label is
```

Comment:
`the `cpumanager` label is...`  Can we elaborate where is this label? It's explained in the Procedure section, but here is the first mention of the `cpumanager` label so maybe it should be here?

Reply:
Hi @rhrazdil - Check out this version. 

http://file.bos.redhat.com/bgaydos/042220/cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.html

http://file.bos.redhat.com/bgaydos/042220/cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.html

I changed the wording in the 'about' file to match the usage in the procedure sections below. Let me know if this version is more clear. I believe I was leaving out the aspect of the node in relation to the label and policy before, which is why it was confusing.

Thanks

Bob

Reply:
Yes, it is more clear now. Thanks